### PR TITLE
Judges!

### PR DESCRIPTION
**Pull request by aepyornis**
Adds the ability to create, edit and remove judges.

_/admin/judges_ -> view a list of judges
_/admin/judges/id/edit_ -> edit a judge
_/admin/judges/id/new_ -> add a new judge

### CODE REVIEW

**Comment by aepyornis** [comment]
not sure why the circleci thingy failed...the tests passed locally me for...i swear!

**Comment by latazzajones** [comment]
@aepyornis @CZagrobelny I'm getting a lot of the same failures on ci 🤷‍♀️

File: app/views/admin/judges/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<div class="row">
+    <div class="col-md-3">
+	<h1>Judges</h1>
+    </div>
+    <div class='col-md-4 col-md-offset-5'>
+	<!-- search box? -->
```
Comment (CZagrobelny) [inline app/views/admin/judges/index.html.erb]: We can skip the search box, it'll be a short list!

**Comment by CZagrobelny** [comment]
@latazzajones @aepyornis I re-ran the tests on circle and they are passing.  I created a card to look into these 6 tests that keep failing on circle.

File: app/views/admin/judges/index.html.erb (hunk 1 above)
Comment (aepyornis) [inline app/views/admin/judges/index.html.erb]: makes sense to me!

**Comment by CZagrobelny** [comment]
@aepyornis this looks great, made a few styling changes for consistency across the app and merging!  Thank you!!!!

**Pull request merged**